manifests/backuphost.pp

```diff
@@ -3,7 +3,7 @@
   String        $chroot,
   String        $mountpoint,
   String        $user   = 'backup',
-  Array[String] $allow_clients = hiera('backup_pfx'),
+  Array[String] $allow_clients = lookup('backup_pfx', Array[String], undef, ['130.242.125.68', '130.242.121.73']),
```

Comment:
I don't know if this is controversial or not, I don't want to lock out anyone from ssh, so set hoppjerka and joppherka here as default